develop/2017-04-29-fabiocorreacordeiro-Local-Transportation.py

```diff
@@ -0,0 +1,568 @@
+
+# coding: utf-8
+
+# # Exploratory Descriptive Analysis of Local Transportation
+# 
+# This is an Exploratory Descriptive Analysis of expenditure with local transportation. We already created three separated notebooks to explore the data of the subquotas 'Taxi, toll and parking', 'Automotive vehicle renting or charter' and 'Fuels and lubricants'. In this notebook, we will join the analysis of this subquotas.
+# 
+# The subquota 'Automotive vehicle renting or charter' changed in the time. It started like 'Automotive vehicle renting or watercraft charter' and in October/2013 was split in 'Watercraft renting or charter' and 'Automotive vehicle renting or charter'. 
+# 
+# The previous notebooks are in:
+# 
+# 'Automotive vehicle renting or charter': https://github.com/datasciencebr/serenata-de-amor/pull/206/commits/6afda4344c6565ec01536242353730b14f2ebaee

```

Comment:
Two things here.

First, to properly format this _list_ of links in your notebook use the following notation:

```
* [Automotive vehicle renting or charter](https://github.com/datasciencebr/serenata-de-amor/pull/206/commits/6afda4344c6565ec01536242353730b14f2ebaee)
* [Taxi, toll and parking](https://github.com/datasciencebr/serenata-de-amor/blob/master/develop/2017-03-15-fabiocorreacordeiro-taxi-descriptive-analysis.ipynb)
* [Fuels and lubricants](https://github.com/datasciencebr/serenata-de-amor/pull/203/commits/2635528e4faa9d94562e08972164ddba2173b15d)
```
Or, maybe even better: use this `[title](url)` markup to link the notebooks in the first paragraph and you don't need this list ; )

2. Only link notebooks that are either on `master` branch or un this PR. You're linking to specific commits, that's not a best practice I'd say.